controls/listview/grouping/delegate-group-descriptor.md

```diff
@@ -10,12 +10,12 @@ tags: group, radlistview, groupdescriptor
 
 # .NET MAUI ListView Delegate Group Descriptor
 
-DelegateGroup descriptor enables you to group by a custom key (e.g. some complex expression combining two or more properties) instead of being limited by the value of a single property. This descriptor exposes the following properties:
+DelegateGroup descriptor enables you to group by a custom key (for example some complex expression combining two or more properties) instead of being limited by the value of a single property. This descriptor exposes the following properties:
```

Comment:
```suggestion
DelegateGroup descriptor enables you to group by a custom key (for example, some complex expression combining two or more properties) instead of being limited by the value of a single property. This descriptor exposes the following properties:
```